Makefile

```diff
@@ -21,8 +21,10 @@ API_GO_HEADER_FILE:=pkg/apis/header.go.txt
 
 # Container image-related variables
 DOCKERFILE=Dockerfile
-IMAGE_TAG=openshift/origin-cluster-node-tuning-operator
-IMAGE_REGISTRY=quay.io
+REGISTRY=quay.io
+ORG=openshift
+TAG=$(shell git branch | grep \* | cut -d ' ' -f2)
+IMAGE=${REGISTRY}/${ORG}/origin-cluser-node-tuning-operator:${TAG}
```

Comment:
Why are we using ${} rather than $() to refer to `Makefile` variables here?

Reply:
either is a valid reference, if you prefer I can change it.

Reply:
For *consistency* within the `Makefile`, please let's use `$()`